modules/ROOT/pages/about-logging-of-payload-formats.adoc

```diff
@@ -85,13 +88,17 @@ The Runtime Manager agent only passes on payload messages where the *Converted t
 |	|Multipart|	"multipart/mixed, multipart/alternative, multipart/digest multipart/parallel"	|FALSE||
 |===
 
-== Customizing Conversion Behavior
+== Customize Conversion Behavior
+
+You can override the default behavior and configure the agent to forward all payload formats or none by adding
+properties to the `conf/mule-agent.yml` file.
```

Comment:
ditto here: it's a file name, distinct from the narrative by its very format. Why format it as if it were syntax?
OK...I'll stop....

Reply:
The formatting guidelines when I first joined were to format filenames (and URLs) as code - there was a proposal that Mysti put together. I can't find it now and I'm not sure it got migrated.

I feel less strongly about this font change, but some filenames are not as obvious, so...